core-bundle/tests/Contao/IdnaTest.php

```diff
@@ -23,21 +23,13 @@ public function testEncodesUnicodeDomain(): void
         $this->assertSame('xn--fbar-5qaa.de', Idna::encode('fööbar.de'));
         $this->assertSame('', Idna::encode(''));
         $this->assertSame('', Idna::encode(sprintf('f%sbär.de', str_repeat('o', 53))));
-
-        $this->expectException(DomainOutOfBoundsException::class);
```

Comment:
Use of `DomainOutOfBoundsException` not used anymore.

Reply:
Fixed in ad6f9d55048fc2bff92afd61989b275430ef18fe.